contrib/stack/stripmapStack/cropFrame.py

```diff
@@ -161,18 +161,18 @@ def cropFrame(frame, limits, outdir, israw=False):
     ####sensing start
     ymin = np.floor( (limits[0] - frame.sensingStart).total_seconds() * frame.PRF)
     print('Line start: ', ymin)
-    ymin = np.int( np.clip(ymin, 0, frame.numberOfLines-1))
+    ymin = int( np.clip(ymin, 0, frame.numberOfLines-1))
 
 
     ####sensing stop 
     ymax = np.ceil( (limits[1] - frame.sensingStart).total_seconds() * frame.PRF) + 1
     print('Line stop: ', ymax)
-    ymax = np.int( np.clip(ymax, 1, frame.numberOfLines)) 
+    ymax = int( np.clip(ymax, 1, frame.numberOfLines)) 
 
     print('Line limits: ', ymin, ymax)
     print('Original Line Limits: ', 0, frame.numberOfLines)
 
-    if (ymax-ymin) <= 1:
+    if (ymax-ymin) < 0:
```

Comment:
why do you want to allow zero? Does zero mean that there is one pixel overlap?

Reply:
ping @ssangha, I think this is the only part we'd like clarification about. LGTM otherwise.